hack/source-manifests.sh

```diff
@@ -25,9 +25,12 @@ if [ -z "$NOOBAA_IMAGE" ] || [ -z "$ROOK_IMAGE" ]; then
 fi
 
 # always start fresh and remove any previous artifacts that may exist.
+rm -rf $OCS_FINAL_DIR
 rm -rf $OUTDIR_TEMPLATES
 mkdir -p $OUTDIR_TEMPLATES
 mkdir -p $OUTDIR_CRDS $OUTDIR_BUNDLEMANIFESTS
+cp deploy/bundlemanifests/*.yaml $OUTDIR_BUNDLEMANIFESTS/
+rm deploy/bundlemanifests/*.yaml
```

Comment:
I think this is the main trick explained in the commit message. Since we are trying to trick the operator-sdk (if I understood it correctly), it might be good to add an explaning comment here (and at the bottom when we copy it back), since this is not obvious from reading the code.